sunpy/net/attr.py

```diff
@@ -1,11 +1,4 @@
 # -*- coding: utf-8 -*-
-# Author: Florian Mayer <florian.mayer@bitsrc.org>
-#
-# This module was developed with funding provided by
```

Comment:
I think we should keep the funding comment. Given we have others in the code base??

Reply:
Start that doc now ;)

Reply:
Sure, ill dump it somewhere inthe docs until we figure out a better place in a different PR, once I butcher the docs.